.changeset/little-llamas-kick.md

```diff
@@ -0,0 +1,8 @@
+---
+'@crowdstrike/glide-core': patch
+---
+
+- Dropdown no longer throws when its default slot contains both no options and no whitespace.
+- Dropdown's "No Results Found" copy, shown when every option is hidden via filtering, has been changed to "No matching options".
+- Dropdown now displays "No options available" when open and its default slot is empty.
+- Dropdown's `filter()` method can now be overridden to return `void`.
```

Comment:
@nicolechung: Added this for you. No need to return an empty array after we publish a new version and you upgrade.